src/main/java/forestry/farming/logic/FarmLogicCocoa.java

```diff
@@ -193,40 +195,44 @@ private Collection<ICrop> getHarvestBlocks(Vect position) {
         return crops;
     }
 
-    private ArrayList<Vect> processHarvestBlock(Stack<ICrop> crops, Set<Vect> seen, Vect start, Vect position) {
+    private ArrayList<Vect> processHarvestBlock(Deque<ICrop> crops, Set<IVect> seen, Vect start, IVect position) {
```

Comment:
👍 (read previous comment)

Reply:
I actually missed this one, had tried to set all `Set<IVect>` back to `Set<Vect>` for the reasons mentioned.